tools/transform.cpp

```diff
@@ -506,6 +506,16 @@ static uint64_t max_gep(const Instr &inst) {
       max_mem_access = UINT64_MAX;
     return 0;
   }
+  if (auto slen = dynamic_cast<const Strlen *>(&inst)) {
+    auto cop = dynamic_cast<ConversionOp *>(slen->getPointer());
+    auto glb = cop && cop->getOp() == ConversionOp::BitCast ?
+        dynamic_cast<GlobalVariable *>(&cop->getValue()) : nullptr;
```

Comment:
maybe hoist this code into a function since it seems it will be helpful for other things beyond strlen.